include/diaspora.php

```diff
@@ -161,6 +162,56 @@ private static function verify_magic_envelope($envelope) {
 	}
 
 	/**
+	 * @brief encrypts data via AES
+	 *
+	 * @param string $key The AES key
+	 * @param string $iv The IV (is used for CBC encoding)
+	 * @param string $data The data that is to be encrypted
+	 *
+	 * @return string encrypted data
+	 */
```

Comment:
I've just tested and can decrypt rino=1 with

    openssl_decrypt($str,'aes-128-ecb',$key,OPENSSL_RAW_DATA,openssl_random_pseudo_bytes(16,$tt));

and it works fine. Encryption should be the same. Note that this only works because Friendica *always* uses a 16-byte key and there's no need to pad it. If you use aes_encrypt()/aes_decrypt() with any other length of key you'll need to copy the key padding hacks to the updated functions. Don't worry about $tt - if you're using rino=1 you probably aren't likely to care about the quality of openssl's random number generator and there's not a lot you can do about it anyway.